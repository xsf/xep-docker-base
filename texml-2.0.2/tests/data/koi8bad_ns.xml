<?xml version="1.0" encoding="koi8-r"?>
<TeXML>Плохие символы: &#1917;&#2004;</TeXML>
