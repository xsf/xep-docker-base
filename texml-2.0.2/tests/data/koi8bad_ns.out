Плохие символы: \unicodechar{1917}\unicodechar{2004}
